webboot/webboot.go

```diff
@@ -23,9 +40,88 @@ var (
 	ipv6    = flag.Bool("ipv6", true, "use IPV6")
 )
 
+func wgetArgs(argURL string) (read io.ReadCloser, outFile string, werr error) {
+	//checks if argURL is a key in osSystems map
+	if _, ok := osSystems[argURL]; !ok {
+		//if argURL is not the map, use it as a URL link for wget
+		read, werr = wget(argURL)
+	} else {
+		// if argURL is in osSystems map, use its download link for wget
+		read, werr = wget(osSystems[argURL].DownloadLink)
+		outFile = argURL
+	}
+	return read, outFile, werr
+}
+
+func wget(arg string) (io.ReadCloser, error) {
```

Comment:
could you use some name other than arg now? We know it's an arg; could it be URL or something like that? 
// wget takes a URL and returns an io.ReadCloser for it and an error, if any. 
// Errors include a failure of http.Get or a returned status code being anything other than 200.

(and what is 200? is there a definition of it in any package?

Reply:
In the http package details, they show the constants for the package. One of the constants include 200, which is StatusOK. They show many other status number constants assigned to variables that explain briefly what the numbers mean and stand for. However, I added a comment right above this line of code inside my file to explain what status 200 stands for and why it is needed to checked.